pandas/_libs/groupby_helper.pxi.in

```diff
@@ -332,7 +343,8 @@ def get_dispatch(dtypes):
 def group_last_{{name}}(ndarray[{{dest_type2}}, ndim=2] out,
                         ndarray[int64_t] counts,
                         ndarray[{{c_type}}, ndim=2] values,
-                        ndarray[int64_t] labels):
+                        ndarray[int64_t] labels,
+                        Py_ssize_t min_count):
```

Comment:
you can set == -1 here (and actually all routines) i think

Reply:
OK, will do (had to for OHLC). I couldn't remember if cython took a perf hit for checking default values.